types/react-helmet/index.d.ts

```diff
@@ -48,6 +48,11 @@ export interface HelmetDatum {
     toComponent(): React.Component<any, any>;
 }
 
+export interface HelmetAttributesTagObject {
+    toString(): string;
+    toComponent(): React.HTMLAttributes<HTMLBodyElement | HTMLElement>;
+}
+
```

Comment:
That sound reasonable to me. Also I do not  use the library before so if you would like to give this PR a try and let me. I would greatly appreciate it!